sds_data_manager/lambda_code/SDSCode/create_schema.py

```diff
@@ -0,0 +1,95 @@
+import json
+import os
+
+import boto3
+import psycopg2
+import requests
+from psycopg2 import Error
+
+# TODO: The code in this lambda will be updated in the next PR
+# to reflect the schema that will be needed by the SDC.
+# Its current state is the bare minimum to get a table
+# created to allow files to be indexed.
+
+# Add code to get Secret instead
+s3 = boto3.client("s3")
+
+
+def send_response(event, context, response_data, response_status):
+    response_url = event["ResponseURL"]
+    response_body = {
+        "Status": response_status,
+        "Reason": "See the details in CloudWatch Log Stream: "
+        + context.log_stream_name,
```

Comment:
cool. I didn't know you could get those values from `context`. That input has been a mystery for me and never saw it used.

Reply:
Yeah, I had no idea what that was before this either. You can get a few things from that object, like the name of the function, the resource arn that invoked the lambda, memory, etc. : https://docs.aws.amazon.com/lambda/latest/dg/python-context.html